work-scheduler-data/pom.xml

```diff
@@ -39,6 +39,14 @@
             <groupId>io.craftsman</groupId>
             <artifactId>dozer-jdk8-support</artifactId>
         </dependency>
+        <dependency>
```

Comment:
The Java Standard Tag Library (JSTL) should not be part of your Data Module.
It is only related to the web layer.